cs481-ekh/f21-welolo
# Task #25

Verification Steps:
1) Run both server and client
2) Enter a wrong phone number
3) Notice the error message on the browser
4) Open the developer tools and check the error on the Network Response tab
